packages/expo-updates/android/src/main/java/expo/modules/updates/loader/LoaderTask.java

```diff
@@ -32,7 +32,8 @@
 
   public interface LoaderTaskCallback {
     void onFailure(Exception e);
-    void onManifestLoaded(Manifest manifest);
+    boolean onCachedUpdateLoaded(UpdateEntity update);
```

Comment:
Document what the return type means since that behavior isn't communicated in the method name.